i18n/en.json

```diff
@@ -2016,6 +2016,14 @@
     "id": "api.license.request-trial.bad-request.terms-not-accepted",
     "translation": "You must accept the Mattermost Software Evaluation Agreement and Privacy Policy to request a license."
   },
+  {
+    "id": "api.license.request-trial.can-start-trial.error",
+    "translation": "Could not check if a trial can be started"
+  },
+  {
+    "id": "api.license.request-trial.can-start-trial.not-allowed",
+    "translation": "TODO Not Allowed"
```

Comment:
Just make sure we put the correct text here

Reply:
Done.